app/models/journeys/page_sequence.rb

```diff
@@ -119,5 +119,13 @@ def claim_submittable?
     def journey_ineligible?
       @journey_ineligible ||= journey::EligibilityChecker.new(journey_session: @journey_session).ineligible?
     end
+
+    def completed?
+      if journey == Journeys::AdditionalPaymentsForTeaching
```

Comment:
Is TRN not required for student loans now?

Is there a reason this can't just be `claim_submittable?`. The claim submission form (which `claim_submittable?` checks) constructs an eligibility* and checks if it's valid so if the eligibility requires a TRN then `claim_submittable?` will return false if the TRN isn't set, so we don't need to check the answers for the presence of a TRN. If the eligibility doesn't require the TRN to be set then `claim_submittable` will handle that too. I might be missing something, in which case this is fine.


*(I actually want to change this so it just uses the policy checkers and only creates an eligibility on save but that's a separate matter!)